### ISSUE TITLE
Issue while executing yotta target oshchip-gcc

### ISSUE THREAD

**Issue by prashantrar** [drewcassidy/target-OSHChip-gcc#1]
Hello,

I have successfully installed yotta on my ubuntu system but when i try to initialize the target or install mbed drivers, this is the error i get.

info: get versions for oshchip-gcc
Fatal Exception, yotta=unknown
Traceback (most recent call last):
  File "/usr/local/bin/yotta", line 4, in <module>
    yotta.main()
  File "/usr/local/lib/python2.7/dist-packages/yotta/main.py", line 201, in main
    status = args.command(args, following_args)
  File "/usr/local/lib/python2.7/dist-packages/yotta/target.py", line 132, in execCommand
    target, errors = c.satisfyTarget(t)
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/component.py", line 677, in satisfyTarget
    shrinkwrap = self.getShrinkwrap()
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/target.py", line 164, in getDerivedTarget
    inherit_shrinkwrap = shrinkwrap
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/access.py", line 381, in satisfyVersion
    name, version_required, working_directory, type=type, inherit_shrinkwrap = inherit_shrinkwrap
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/access.py", line 314, in satisfyVersionByInstalling
    v = latestSuitableVersion(name, version_required, _registryNamespaceForType(type))
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/access.py", line 107, in latestSuitableVersion
    vers = remote_component.availableVersions()
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/registry_access.py", line 485, in availableVersions
    return _listVersions(self.namespace, self.name)
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/registry_access.py", line 100, in wrapped
    return fn(_args, *_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/registry_access.py", line 232, in _listVersions
    request_headers = _headersForRegistry(registry)
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/registry_access.py", line 414, in _headersForRegistry
    'X-Yotta-Client-Version': _getYottaVersion(),
  File "/usr/local/lib/python2.7/dist-packages/yotta/lib/registry_access.py", line 397, in _getYottaVersion
    _yotta_version = pkg_resources.require("yotta")[0].version
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 725, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 628, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: ndg-httpsclient

**Comment by drewcassidy** [comment]
Does this happen with other targets?

**Comment by prashantrar** [comment]
Yes for all targets, even when i try to install mbed drivers using yotta i get the same issue

**Comment by drewcassidy** [comment]
I'm going to close this issue because it's not specific to the OSHChip target. I suggest you open an issue on the Yotta github page [here](https://github.com/ARMmbed/yotta).

**Comment by bitbangr** [comment]
I may have had an similar issue with Yotta using the Mac App installer. I could not search or set targets, build did not work etc.  

As the last line of the above error is "ndg-httpsclient" I am guessing this may have something to do with SSL Certificates. 

I opened this ticket [Invalid SSL Certificate for https://yottabuild.org is preventing yotta build from working](https://github.com/ARMmbed/yotta/issues/744)

It was suggested that I do a new install of the latest version of Yotta from the master version.  
I did this and everything seems to work fine now.

Mark
